packages/web-react/src/components/Modal/useModalStyleProps.ts

```diff
@@ -0,0 +1,34 @@
+import { useClassNamePrefix } from '../../hooks';
```

Comment:
Please add also a test for this hook. It should be a simple copy of existing tests. Thanks.